### ISSUE TITLE
API ID e Hash não ficam mais salvos

### ISSUE THREAD

**Issue by EduFerSouz** [apenasrr/clonechat#23]
Hoje começou a pedir API ID e Hash da conta toda vez que abro o prompt do Clonechat, e pede pra deletar o arquivo de sessão. É algum bug aqui comigo ou é algo que o Telegram mudou e causou isso? Até dois dias atrás estava normal.
![clone](https://github.com/apenasrr/clonechat/assets/109004011/617e306a-6af0-46c8-80e3-36d0e0784e10)

**Comment by matheusbach** [comment]
Delete Session file and  try again.

**Comment by EduFerSouz** [comment]
> Delete Session file and  try again.

Yes, I need to do this every time I will use the script. The session don't still save anymore, I have to put API ID and hash always now.
